packages/types/src/profiling.ts

```diff
@@ -50,7 +50,6 @@ export interface ContinuousThreadCpuProfile {
 }
 
 interface BaseProfile<T> {
-  timestamp: string;
```

Comment:
We don't need this anymore?

Reply:
It's already defined on the txn format, so I just removed it so that the chunk one doesnt inherit from it